source/user-manual/api/configuration.rst

```diff
@@ -347,6 +349,18 @@ agents
 | allow      | yes, true, no, false | true          | Allow uploading configurations that accept higher agent versions through the Wazuh API. Setting this option to ``false`` prevents uploading ``ossec.conf`` files that contain the ``<allow_higher_versions>`` section with the ``yes`` value inside the :ref:`auth <reference_ossec_auth>` or :ref:`remote <reference_ossec_remote>` tags.  |
 +------------+----------------------+---------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+indexer
+~~~~~~~
+
+.. versionadded:: 4.8.0
+
++------------+----------------------+---------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Sub-fields | Allowed values       | Default value | Description                                                                                                                                                                                                                                                                                                                                 |
++============+======================+===============+=============================================================================================================================================================================================================================================================================================================================================+
+| allow      | yes, true, no, false | true          | Allow uploading configurations with modified indexer values through the Wazuh API. Setting this option to ``false`` prevents uploading ``ossec.conf`` files if the ``<indexer>`` section has changed.                                                                                                                                       |
```

Comment:
I would add a link to the `indexer` tag reference (`user-manual/reference/ossec-conf/indexer.html`).